syntax = "proto3";
package c2gs;
import "common.proto";
option cc_generic_services = true;

message EnterSceneRequest
{
	uint64 player_id  = 1;
	uint64 scene_confid = 2;
}

message EnterSceneResponse
{
	ClientError error = 1;
	SceneInfo scenes_info = 2;
}

service C2GsService {
  rpc EnterScene (EnterSceneRequest) returns (EnterSceneResponse);
}